PathWeaver/Paths/Unnamed.path

```diff
@@ -0,0 +1,3 @@
+X,Y,Tangent X,Tangent Y,Fixed Theta,Name
+29.94274677285252,-96.93435970572774,9.930715178311477,0.0,true,
+162.0212586443952,-107.85814640187039,0.9930715178311686,-62.56350562336229,true,
```

Comment:
This file shouldn't be in the commit. You have it in the gitignore. Run `git rm -r --cached .` before your next commit to reapply the gitingore.